sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerSchema.java

```diff
@@ -169,7 +169,9 @@ private static Type parseSpannerType(String spannerType) {
       if ("DATE".equals(spannerType)) {
         return Type.date();
       }
-
+      if ("NUMERIC".equals(spannerType)) {
+        return Type.numeric();
+      }
```

Comment:
@terryxian78 Is this new?

Reply:
No, This was included in my original pull request, I explained in the comment.

Reply:
Can we move this to a separate PR with tests ?

Reply:
sure. I will make it a separate PR.

Reply:
Done! I reverted the change of this file.